appsync/index.js

```diff
@@ -136,22 +136,73 @@ async function getDetailsForVehicle(userId, vehicleId, authToken) {
     accelerationScore: tripSummaryData.accelerationScore,
     totalScore: tripSummaryData.score,
     travelDistanceThisYear: 100,
-    fuelLeft: 999,
     parking: recentTrip.location,
     timeTraveled: tripSummaryData.TODO,
     trips: tripsForVehicle,
     lifeLitresPerHundredKm: tripsForVehicle,
     recentTrip: recentTrip,
-    parkedVehicle: parkedUserVehicles
+    parkedVehicle: parkedUserVehicles,
+    fuelLeft: fuelLeft.kmsLeft,
+    averagePer100Km: fuelLeft.averagePer100Km,
+    litresLeft: fuelLeft.litresLeft
   };
   return finalResult;
 }
 
+const calculateFuelLeft = (refillData, trips) => {
```

Comment:
This is declared after it is used. Would suggest you move it up above where it is called on line 115